src/main/kotlin/app/revanced/patches/all/ads/blocklist/AndroidAdsBlocklist.kt

```diff
@@ -0,0 +1,31 @@
+package app.revanced.patches.all.ads.blocklist
```

Comment:
This class isn't necessarily required; you can put it in the companion object of the patch that uses this class.